File: Modul_5_kel41/userService.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Modul_5_kel41
{
    class userService
    {
        private string[,] data;
        private string email, password, roles = "";

        public userService(string emails, string passwords)
        {
            email = emails;
            password = passwords;
            data = new string[2, 3] {
                {"hanaadilah@gmail.com", "12345", "superadmin" },
                {"gunawanprasetya@gmail.com", "12345", "user"  }
            };
        }

        public void login()
        {
            var (status, role) = checkCredentials();
            if (status == true)
            {
                Console.WriteLine("\nWelcome " + role);
                Console.WriteLine("Logged it as user email: " + email);
            }
            else
            {
                Console.WriteLine("\nInvalid Login");
            }
        }
        private (bool, string) checkCredentials()
        {
            for (int i = 0; i < data.GetLength(0); i++)
            {
                if (data[i, 0] == email && data[i, 1] == password)
                {
                    roles = data[i, 2];
                    return (true, roles);
                }
            }
            return (false, roles);
        }
    }
}

File: Modul5/userService.py
class userService(object):
    def __init__(self, email, password):
        self.email = email
        self.password = password
        self.data = {
            "HanaAdilah@gmail.com" : {
                "email"     : "HanaAdilah@gmail.com",
                "password"  : "12345",
                "role"      : "superadmin"
            },
            "GunawanPrasetya@gmail.com" : {
                "email"     : "GunawanPrasetya@gmail.com",
                "password"  : "12345",
                "role"      : "user"
            }
        }
    def login(self):
        get_data = self.checkCredentials()
        if get_data:
            print("\nWelcome ",get_data['role'])
            print("Logged it as user email: ",get_data['email'])
        else:
            print("\nInvalid Login!\n")

    def checkCredentials(self):
        for value in self.data:
            if value == self.email:
                get_data_user = self.data[value]
                if self.password == get_data_user['password']:
                    return get_data_user
                else:
                    return False
